components/malloc_size_of/lib.rs

```diff
@@ -782,3 +791,91 @@ impl MallocSizeOf for xml5ever::QualName {
         self.local.size_of(ops)
     }
 }
+
+impl MallocSizeOf for hyper::header::Headers {
+    fn size_of(&self, ops: &mut MallocSizeOfOps) -> usize {
+        let mut t = 0;
```

Comment:
`self.iter().fold(0, |acc, x| acc + x.size_of(ops))`

Reply:
For this, I followed the suggestion for when we could not implement MallocSizeOf for HeaderView! But, on quick glance, it does look like I could fit it with fold!